.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,7 +10,7 @@ Screenshots before and after (if PR changes UI):
 -
 -
 
-How did I test this code:
+What tests were done:
```

Comment:
How about _What did I do to test the code and ensure quality?_?

Reply:
Sounds good.